data/json/construction.json

```
@@ -3843,7 +3843,7 @@
     "description": "Build Plastic Window",
     "//": "Step 2: Build curtain for plastic window",
     "category": "WINDOWS",
-    "required_skills": [ [ "fabrication", 7 ] ],
+    "required_skills": [ [ "fabrication", 2 ] ],
```

```suggestion
    "required_skills": [ [ "fabrication", 1 ] ],
```
Maybe? If you go with my description?